packages/vetrina/src/Vetrina/Products.purs

```diff
@@ -0,0 +1,63 @@
+module Vetrina.Products where
+
+import Prelude
+
+import Data.Maybe (Maybe, maybe)
+import Effect (Effect)
+import React.Basic (JSX, make)
+import React.Basic as React
+import React.Basic.DOM as DOM
+import React.Basic.Events (handler_)
+import Vetrina.Types (Product)
+
+type Self = React.Self Props State
+type State = {}
+type Props =
+  { products        :: Array Product
+  , onProductChange :: Product -> Effect Unit
+  , selectedProduct :: Maybe Product
+  }
+
+component :: React.Component Props
+component = React.createComponent "Products"
+
+product :: Props -> JSX
+product = make component
+  { initialState: {}
+  , render
+  }
+
+render :: Self -> JSX
+render self =
+  DOM.div
+    { className: "products--container"
+    , children:  map renderProduct self.props.products
+    }
+  where
+    renderProduct p =
+      DOM.label
+        { className: "vetrina--product"
+        , children: [ DOM.input $
```

Comment:
The `InputField` component exists mainly because of the styling. The products we show in Vetrina are very unique in design.

Should have attached this image in the description... Anyway, this is roughly what we want (only some texts and the background color of the button have changed.)
![image (3)](https://user-images.githubusercontent.com/6303323/93068059-ac984e00-f684-11ea-9017-7d1d10a10597.png)